docs/install/installing-istio.md

```diff
@@ -203,7 +203,7 @@ Knative dispatches to different services based on their hostname, so it is recom
 To do this, begin by looking up the external IP address that Istio received:
 
 ```
-$ kubectl get svc -nistio-system
+$ kubectl get svc -n istio-system
```

Comment:
It is alright to change, but it is actually not an issue. `-nistio-system` is still valid.

Reply:
True. `kubectl get svc -nistio-system` works just fine. Just to avoid confusion.